client/src/controllers/RevealController.ts

```diff
@@ -219,6 +228,35 @@ export class RevealController extends Controller<HTMLElement> {
       });
   }
 
+  setSavedClosedValue(closed) {
+    const id = this.element.id;
+
+    if (!this.saveStateValue || !id) return;
+
+    try {
+      localStorage.setItem(
+        `wagtail:reveal-${id}-closed`,
```

Comment:
```suggestion
        `wagtail:${this.identifier}-${this.element.id}-closed`,
```

This way we can easily use this for when registered as different identifiers. With cleaner storage keys.